scripts/cnode-helper-scripts/deploy-as-systemd.sh

```diff
@@ -376,6 +376,45 @@ else
   echo "cncli executable not found... skipping!"
 fi
 
+echo
```

Comment:
Since there is an option to use outside of `env`, might make more sense to use -d flag on script to deploy as systemd, and then call it in deploy-as-systemd?
(eg: [here](https://github.com/cardano-community/guild-operators/blob/alpha/scripts/cnode-helper-scripts/cnode.sh#L63-L103) )

Reply:
ok -d and -s for internal deployment and service-mode is now implemented